.changeset/sixty-hounds-reflect.md

```diff
@@ -0,0 +1,9 @@
+---
+'@theguild/tailwind-config': minor
+---
+
+- use ESM config for `postcss`
+- use `postcss-lightningcss` instead of `autoprefixer` and `cssnano`
```

Comment:
TIL about lightningcss

Reply:
😁 I used `postcss-lightningcss` in Nextra previously